app-emulation/simh/simh-4.0.0_git.recipe

```diff
@@ -99,23 +116,73 @@ BUILD_PREREQUIRES="
 	dejavu
 	"
 
+defineDebugInfoPackage simh$secondaryArchSuffix \
+	"$commandBinDir"/3b2$commandSuffix \
+	"$commandBinDir"/altair$commandSuffix \
+	"$commandBinDir"/altairz80$commandSuffix \
+	"$commandBinDir"/b5500$commandSuffix \
+	"$commandBinDir"/cdc1700$commandSuffix \
+	"$commandBinDir"/eclipse$commandSuffix \
+	"$commandBinDir"/gri$commandSuffix \
+	"$commandBinDir"/h316$commandSuffix \
+	"$commandBinDir"/hp2100$commandSuffix \
+	"$commandBinDir"/hp3000$commandSuffix \
+	"$commandBinDir"/i1401$commandSuffix \
+	"$commandBinDir"/i1620$commandSuffix \
+	"$commandBinDir"/i650$commandSuffix \
+	"$commandBinDir"/i701$commandSuffix \
+	"$commandBinDir"/i7010$commandSuffix \
+	"$commandBinDir"/i704$commandSuffix \
+	"$commandBinDir"/i7070$commandSuffix \
+	"$commandBinDir"/i7080$commandSuffix \
+	"$commandBinDir"/i7090$commandSuffix \
+	"$commandBinDir"/i7094$commandSuffix \
+	"$commandBinDir"/ibm1130$commandSuffix \
+	"$commandBinDir"/id16$commandSuffix \
+	"$commandBinDir"/id32$commandSuffix \
+	"$commandBinDir"/imds-225$commandSuffix \
+	"$commandBinDir"/isys8010$commandSuffix \
+	"$commandBinDir"/isys8020$commandSuffix \
+	"$commandBinDir"/isys8024$commandSuffix \
+	"$commandBinDir"/isys8030$commandSuffix \
+	"$commandBinDir"/lgp$commandSuffix \
+	"$commandBinDir"/microvax1$commandSuffix \
+	"$commandBinDir"/microvax2$commandSuffix \
+	"$commandBinDir"/microvax3900$commandSuffix \
+	"$commandBinDir"/nova$commandSuffix \
+	"$commandBinDir"/pdp1$commandSuffix \
+	"$commandBinDir"/pdp10$commandSuffix \
+	"$commandBinDir"/pdp11$commandSuffix \
+	"$commandBinDir"/pdp15$commandSuffix \
+	"$commandBinDir"/pdp4$commandSuffix \
+	"$commandBinDir"/pdp7$commandSuffix \
+	"$commandBinDir"/pdp8$commandSuffix \
+	"$commandBinDir"/pdp9$commandSuffix \
+	"$commandBinDir"/rtvax1000$commandSuffix \
+	"$commandBinDir"/s3$commandSuffix \
+	"$commandBinDir"/scelbi$commandSuffix \
+	"$commandBinDir"/sds$commandSuffix \
+	"$commandBinDir"/ssem$commandSuffix \
+	"$commandBinDir"/swtp6800mp-a$commandSuffix \
+	"$commandBinDir"/swtp6800mp-a2$commandSuffix \
+	"$commandBinDir"/tx-0$commandSuffix \
+	"$commandBinDir"/vax$commandSuffix \
+	"$commandBinDir"/vax730$commandSuffix \
+	"$commandBinDir"/vax750$commandSuffix \
+	"$commandBinDir"/vax780$commandSuffix \
+	"$commandBinDir"/vax8600$commandSuffix
+
 BUILD()
 {
-	# Something wrong with the include and lib detection
-	# Let's do it manually for now.
 	make \
-		VIDEO_CCDEFS="$(sdl2-config --cflags)" \
-		VIDEO_LDFLAGS="$(sdl2-config --libs)" \
 		FONTPATH=$(finddir B_SYSTEM_FONTS_DIRECTORY) \
 		$jobArgs
 }
 
 INSTALL()
 {
-	mkdir -p "$prefix/bin"
+	mkdir -p "$commandBinDir"
 	for bin in BIN/*; do
```

Comment:
I think this would do the same job (but I did not check yet):
```
INSTALL()
{
	install -d "$commandBinDir"
	install -t "$commandBinDir" BIN/*
}
```